front/components/assistant/AssistantBrowser.tsx

```diff
@@ -106,6 +106,11 @@ export function AssistantBrowser({
           ? visibleTabs[0].id
           : null;
 
+  const isVaultsEnabled = owner.flags.includes("data_vaults_feature");
+  const builderManageAssistantsRoute = isVaultsEnabled
+    ? `/w/${owner.sId}/assistant/manage-assistants/`
```

Comment:
I'd say it should be `/w/${owner.sId}/assistants`

Reply:
Agreed it's better, I added a commit!